benchmarks/cugraph/standalone/bulk_sampling/cugraph_bulk_sampling.py

```diff
@@ -564,17 +565,39 @@ def benchmark_cugraph_bulk_sampling(
     output_sample_path = os.path.join(output_subdir, "samples")
     os.makedirs(output_sample_path)
 
-    batches_per_partition = 200_000 // batch_size
+    if sampling_target_framework == "cugraph_dgl_csr":
+        sampling_kwargs = {
+            "deduplicate_sources": True,
+            "prior_sources_behavior": "carryover",
+            "renumber": True,
+            "compression": "CSR",
+            "compress_per_hop": True,
+            "use_legacy_names": False,
+            "include_hop_column": False,
+        }
+    else:
+        # FIXME: Update these arguments when CSC mode is fixed in cuGraph-PyG (release 24.02)
+        sampling_kwargs = {
+          "deduplicate_sources": True,
+          "prior_sources_behavior": "exclude",
+          "renumber": True,
+          "compression": "COO",
+          "compress_per_hop": False,
+          "use_legacy_names": False,
+          "include_hop_column": True
+        }
```

Comment:
Does this setting also work for cugraph-dgl COO code path?

Reply:
I will have to test, I just focussed on `CSC` code path to ensure we have success there (Given it is the fastest one) but I dont see why it will not work.

Reply:
But `prior_sources_behavior` needs to be True for dgl, right? I think COO path would be useful in the future for debugging purposes.

Reply:
@tingyu66 , I have not spent time exploring `COO` code path, do you think I should focus on it , do we expect it to have equivalent speed (maybe after optimizations)  ?

Reply:
I will have to add something like ` sampling_target_framework=='cugraph_dgl_coo' ` with `prior_sources_behavior =True` .  The above path is for `cugraph-pyG` .

Reply:
I don't think getting COO up to speed is a priority, but it would be useful to include/document the parameter combinations needed for COO path, unless we no longer support it.

Reply:
I will add that as a followup:

Filed https://github.com/rapidsai/cugraph/issues/3981 to track.